### DISCUSSION TITLE
Using it with a touch screen

### DISCUSSION THREAD

**Discussion by cgillinger** [Fifteen15Studios/MMM-AutoDimmer#7] [Ideas]
Hi!

I'm doing a project where I use a Raspberry Pi with a touch screen. I'm using Eben Kouao's "Smart Touch" module (https://github.com/EbenKouao/MMM-SmartTouch) to shut it down, and/or reboot it, and to hide the modules. 

However, I'm looking for a way to have a schedule where the screen dims at night, and re-brightens in the morning, and I found your module. But, I would love if I could get the "hide" button from Smart Touch, to instead break the schedule to re-brighten the screen for a fix period, say 30 mins. After that (or by a second press on the button), I'd like it to return to the preprogrammed schedule.

I however suck at coding. Is there a way to achieve this?

**Comment by Fifteen15Studios** [comment]
My suggestion would be to use a module like [MMM-TouchButton](https://github.com/Tom-Hirschberger/MMM-TouchButton) to send a notification, and use that notification to enable and disable the dimming. It looks like each button only send a specified value as a notification, so you could have a dim button and a brighten button, which each send a notification with a different value.
